test/integration-test.py

```diff
@@ -241,6 +245,52 @@ def run_client_tests():
     cmd = os.path.join(root, 'tests', 'boulder-integration.sh')
     run_custom(cmd, cwd=root)
 
+
+def get_future_output(cmd, date, cwd=None):
+    return subprocess.check_output('FAKECLOCK=`date -d "%s"` %s' % (date.isoformat(), cmd), cwd=cwd, shell=True)
+
+def run_expired_authz_purger_test(challenge_types):
+    # Issue the certificate and transform it from DER-encoded to PEM-encoded.
+    exit_code = subprocess.Popen('''
+        node test.js --email %s --agree true \
+          --domains %s \
+          --challType %s --abort-step %s
```

Comment:
We can leave out `--challType` here. And merging abort-step onto the previous line will make the linebreaks a little more consistent.